accelerator-core/src/main/java/com/tokbox/android/otsdkwrapper/wrapper/OTWrapper.java

```diff
@@ -754,6 +757,32 @@ public synchronized void setOutputSignalProtocol(SignalProtocol outputProtocol)
     }
   }
 
+  /**
+   * Return boolean indicating whatever or nor device has multiple cameras available. This allows you to easily
```

Comment:
I'd rewrite this comment to something like
"Returns a boolean indicating whether the device has multiple available cameras."